client/components/editor-standalone/field-definitions/category-field.ts

```diff
@@ -33,7 +33,7 @@ export const getCategoriesField = (): IFieldDefinition => ({
 
         const field: IAuthoringFieldV2 = {
             id: id,
-            name: superdeskApi.localization.gettext('Categories'),
+            name: superdeskApi.localization.gettext('ANPA Category'),
```

Comment:
I think label should stay the same here and `ANPA Category` alias is applied with language overwrites when needed